sdk/schemaregistry/Azure.Data.SchemaRegistry/api/Azure.Data.SchemaRegistry.netstandard2.0.cs

```diff
@@ -34,6 +34,7 @@ public partial class SchemaRegistryClient
         public SchemaRegistryClient(string fullyQualifiedNamespace, Azure.Core.TokenCredential credential) { }
         public SchemaRegistryClient(string fullyQualifiedNamespace, Azure.Core.TokenCredential credential, Azure.Data.SchemaRegistry.SchemaRegistryClientOptions options) { }
         public string FullyQualifiedNamespace { get { throw null; } }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
```

Comment:
I can try to hide this, but I think it's just part of the DPG API surface.

Reply:
It shows up in most libraries when you search the repo - may as well just let it stick around and join the crowd.